scripts/ci/release-latest

```diff
@@ -46,15 +43,19 @@ RELEASE_NAME="${PKG_TARGET}+dcos-ui-${PKG_VERSION}"
 # - dcos-ui/master/dcos-ui-latest
 # - dcos-ui/1.12/dcos-ui-latest
 # - dcos-ui/1.11/dcos-ui-latest
-DCOS_BRANCH="dcos-ui/${RELEASE_NAME}"
+DCOS_BRANCH="dcos-ui/${RELEASE_NAME%+${GIT_COMMIT:0:8}}"
 
 ## Create Assets & Generate Upload Data
 #####################################################################
 
-FORCE_UPLOAD=${FORCE_UPLOAD} \
-  RELEASE_NAME="${RELEASE_NAME}" \
-  ${SCRIPT_PATH}/utils/upload-build
+RELEASE_NAME="${RELEASE_NAME}" \
+  ${SCRIPT_PATH}/utils/upload-build || exit 0
 
+# cleanup all old latest build for this version:
```

Comment:
that snippet gives me some spine shivers. What happens if this goes wrong because the variables are empty, and we  end up deleting the entire contents of dcos-ui? Is this bucket history versioned? I think if this could happen, we might consider taking some preventive steps, like making sure we can revert a bad delete, or adding some test for the FNAME variable so we are sure we are not deleting what we do not want.

Reply:
@juliangieseke please remove this bit as discussed offline.

Reply:
we decided to remove this part again and create a janitor script that removes old builds some time in future. 👍